make audit still available, whereas the corresponding setup command is gone

When the `audit` command was removed (https://github.com/pallets/flask/commit/4fca38c078142744922a8c763bc1468e680a830d), the corresponding shortcut in `Makefile` was not deleted. 

    $ make audit
    
    error: invalid command 'audit'
    make: *** [audit] Error 1

The shortcut was never documented, so it is unlikely that anyone will use it, but still...

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-.PHONY: all install-dev test coverage cov test-all tox docs audit release clean-pyc upload-docs ebook
+.PHONY: all install-dev test coverage cov test-all tox docs release clean-pyc upload-docs ebook
 
 all: test
 
@@ -22,9 +22,6 @@ tox: test-all
 
 docs: clean-pyc install-dev
 	$(MAKE) -C docs html
-
-audit:
-	python setup.py audit
 
 release:
 	python scripts/make-release.py

